splunk-quarkus/src/main/java/com/redhat/console/integrations/ResultTransformer.java

```diff
@@ -1,43 +0,0 @@
-package com.redhat.console.integrations;
-
-import java.util.HashMap;
-import java.util.Map;
-
-import org.apache.camel.Exchange;
-import org.apache.camel.Message;
-import org.apache.camel.Processor;
-
-/**
- * Transformer to create a Map from the outcome of the actual component.
- */
-public class ResultTransformer implements Processor {
```

Comment:
This class was merged with the old `CloudEventEncoder` class and replaced with `OutgoingCloudEventBuilder`.